optaplanner-examples/src/main/java/org/optaplanner/examples/app/OptaPlannerExamplesApp.java

```diff
@@ -199,17 +189,10 @@ private JPanel createDescriptionPanel() {
     private JPanel createExtraPanel() {
         JPanel extraPanel = new JPanel(new GridLayout(0, 1, 5, 5));
         extraPanel.add(new JPanel());
-        webExamplesDialog = new WebExamplesDialog();
-        Action webExamplesAction = new AbstractAction("Show web examples") {
-            @Override
-            public void actionPerformed(ActionEvent event) {
-                webExamplesDialog.setLocationRelativeTo(OptaPlannerExamplesApp.this);
-                webExamplesDialog.setVisible(true);
-            }
-        };
-        extraPanel.add(new JButton(webExamplesAction));
```

Comment:
Good catch.

Can we add links to https://github.com/kiegroup/optaweb-vehicle-routing and https://github.com/kiegroup/optaweb-employee-rostering ? Let's do this in a separate issue and discuss how the UI would look like.

Reply:
OK, I'll just remove the button altogether for now.